content/help/ops/traffic-management/troubleshooting/index.md

```diff
@@ -6,6 +6,37 @@ weight: 30
 
 This section describes common problems and tools and techniques to address issues related to traffic management.
 
+## Requests are rejected by Envoy
+
+Requests may be rejected for various reasons, to identify the reason, you may inspect the access logs. By default,
+access logs are printed into standard output of the container, run the following command to see the log:
+
+{{< text bash >}}
+$ kubectl logs -it PODNAME -c istio-proxy -n NAMESPACE
+{{< /text >}}
+
+In the default access log format, Envoy response flags and Mixer policy status are located after response code,
+if you are using custom log format, make sure to include `%RESPONSE_FLAGS%` and `%DYNAMIC_METADATA(istio.mixer:status)%`.
+
+Refer to the [Envoy response flags](https://www.envoyproxy.io/docs/envoy/latest/configuration/access_log#config-access-log-format-response-flags)
+for details of response flags.
+
+Common response flags are:
+
+- `NR`: No route configured, check your `DestinationRule` or `VirtualService`.
+- `UO`: Upstream overflow with circuit breaking, check your circuit breaker configuration in `DestinationRule`.
+- `UF`: Failed to connect to upstream, if you're using Istio authentication, check your mutual TLS configuration too.
```

Comment:
authentication, check for a [mutual TLS configuration conflict](#503-errors-after-setting-destination-rule).